.github/workflows/publish_to_pypi.yaml

```diff
@@ -31,6 +32,6 @@ jobs:
           .
       - name: Publish distribution 📦 to PyPI
         if: startsWith(github.ref, 'refs/tags')
-        uses: pypa/gh-action-pypi-publish@v1.4.2
+        uses: pypa/gh-action-pypi-publish@76f52bc884231f62b9a034ebfe128415bbaabdfc
```

Comment:
Is it worth adding a comment after the hash of the tag this commit refers to? Just so it can be quickly eyeballed.

Reply:
Updated, and thanks for the quick review!